### PR TITLE
Add IHV and OEM partners page

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add IHV and OEM partners page
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/partners/ihv-and-oem
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](https://docs.google.com/document/d/1p8Et43R14AGiFn9g7ro9VLXZh26gph9Kk5K6_lc8W8U/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2565

## Screenshots

[if relevant, include a screenshot]

### CODE REVIEW

File: templates/partners/ihv-and-oem.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    }
+  }
+</style>
+
+<section class="p-strip is-deep">
+  <div class="u-fixed-width u-align--center">
+    <h4 class="p-muted-heading u-no-max-width">A selection of our Hardware partners</h4>
+  </div>
+  <div class="u-fixed-width">
+    <ul class="p-inline-images u-no-margin--bottom">
```
Comment (deadlight) [inline templates/partners/ihv-and-oem.html:58]: These should be pulled in through a feed, not hard coded

**Comment by waynecrosby** [comment]
@deadlight @solazio - Looking good! Can you make the changes below, the main image in the header is just a placeholder, we will need to purchase before going live.

As it's talking about data centers can we change the laptop image to look like the below:
Partner - IHV   OEM

Temporary image link:
https://assets.ubuntu.com/v1/143fff51-DO+NOT+USE%21.jpg

Screenshot 2020-04-21 at 16 51 07
Can you please change the certificates to the asset link below:
https://assets.ubuntu.com/v1/729c18b3-certificates.svg

**Comment by waynecrosby** [comment]
@deadlight @solazio - Or you could use the illustration below to get it out without the need to purchase the image:


Server icon link:
https://assets.ubuntu.com/v1/0cf81aee-servers-wht.svg

**Comment by waynecrosby** [comment]
@deadlight @solazio - Or you could use the illustration below to get it out without the need to purchase the image:
Partner - IHV   OEM3

Server icon link:
https://assets.ubuntu.com/v1/0cf81aee-servers-wht.svg

**Comment by waynecrosby** [comment]
@solazio Can we please add some more Hardware Partner logos?

**Comment by solazio** [comment]
> @solazio Can we please add some more Hardware Partner logos?

@waynecrosby as they are rendered from the database, we will need to update the database to show up here. And I think the marketing people will have to do that.

**Comment by waynecrosby** [comment]
@solazio Perfect

**Pull request merged**